pkg/resource/aws/testdata/acc/aws_iam_access_key/terraform.tf

```diff
@@ -16,3 +16,13 @@ resource "aws_iam_user" "testuser_access_key" {
 resource "aws_iam_access_key" "accesskey" {
     user = aws_iam_user.testuser_access_key.name
 }
+
+resource "aws_iam_access_key" "ak" {
+    user    = aws_iam_user.test.name
+    pgp_key = "keybase:test"
```

Comment:
Is this related to this PR ? 🤔

Reply:
That's what @sjourdan told us in the last vocal meeting `if you can put it in the refacto, go`. But I can revert the last commit.